frontend/packages/ux-editor/src/components/config/RuleComponent.tsx

```diff
@@ -146,70 +148,62 @@ class Rule extends React.Component<IRuleComponentProps, any> {
+                          )}
+                        </label>
+                        <div className={classes.configureInputParamsContainer} key={key}>
+                          <input
+                            id={paramName}
+                            name={paramName}
+                            type='text'
+                            className={classes.inputType}
+                            value={this.props.ruleModelElements[selectedMethodNr].inputs[key]}
+                            width={10}
+                            disabled={true}
+                          />
+                          <div>
+                            <SelectDataModelComponent
+                              onDataModelChange={this.handleParamDataChange.bind(null, paramName)}
+                              selectedElement={this.state.ruleConnection.inputParams[paramName]}
+                              hideRestrictions={true}
                             />
                           </div>
                         </div>
-                        <div className='col-9 col'>
-                          <SelectDataModelComponent
-                            onDataModelChange={this.handleParamDataChange.bind(null, paramName)}
-                            selectedElement={this.state.ruleConnection.inputParams[paramName]}
-                            hideRestrictions={true}
-                          />
-                        </div>
-                      </div>
+                      </>
                     );
                   }
                 )}
               </div>
-              <div className='form-group a-form-group mt-2'>
-                <h2 className='a-h4'>
+              <div>
+                <h2 className={classes.subTitle}>
                   {this.props.t('ux_editor.modal_configure_rules_configure_output_header')}
                 </h2>
-                {/* length is always 1 since method always returns just one thing */}
-                <div className='align-items-center mt-1 row' key='0'>
-                  <div className='col col-3'>
-                    <div className='form-group a-form-group mt-1 disabled'>
-                      <label className='a-form-label' htmlFor='outParam'>
-                        {this.props.t(
-                          'ux_editor.modal_configure_rules_configure_output_param_helper'
-                        )}
-                      </label>
-                      <input
-                        id='outParam0'
-                        name='outParam0'
-                        type='text'
-                        className='form-control'
-                        value='outParam0'
-                        width={10}
-                        disabled={true}
-                      />
-                    </div>
-                  </div>
-                  <div className='col col-9'>
+                {/* length is always 1 since method always returns just one thing    */}
```

Comment:
I cannot understand the context of this comment, I believe it can be removed. :)

Reply:
I do not know about it actually  because all the code was already in RuleComponent, I just changed in it to communicate with the new css I added.

Reply:
Then I would remove the comment. If we cannot understand it, then there is no reason to have the comment there.